deploy/basic/terraform/scripts/node.sh

```diff
@@ -38,7 +38,7 @@ yum install -y gcc-c++ make nodejs wget unzip httpd jq
 
 yum -y install oracle-release-el7
 yum-config-manager --enable ol7_oracle_instantclient
-yum -y install oracle-instantclient19.3-basic oracle-instantclient19.3-jdbc oracle-instantclient19.3-sqlplus
+yum -y install oracle-instantclient8-basic oracle-instantclient19.8-jdbc oracle-instantclient19.8-sqlplus
```

Comment:
is this right? `instantclient8`?

Reply:
Noticed that after the PR. Was not affecting the usage as the jdbc force to instal the correct basic. Fixed anyways.